src/applications/claims-status/containers/ClaimStatusPage.jsx

```diff
@@ -24,7 +24,8 @@ import { setUpPage, isTab } from '../utils/page';
 class ClaimStatusPage extends React.Component {
   componentDidMount() {
     const { claim } = this.props;
-    setTabDocumentTitle(claim, 'Status');
+    // Only set the document title at mount-time if the claim is already available.
+    if (claimAvailable(claim)) setTabDocumentTitle(claim, 'Status');
```

Comment:
I would have really liked to make all of this more DRY (the changes in this file are repeated nearly verbatim in `<FilesPage>` and `<OverviewPage>`, and the tests for these files suffer similarly), but moving all of this into one of the wrapper components (the `<ClaimPage>` or `<ClaimDetailLayout>`)  felt like it was a bit too far outside the scope of this fix.